testnet/stacks-node/conf/mainnet-follower-conf.toml

```diff
@@ -1,16 +1,16 @@
 [node]
-# working_dir = "/dir/to/save/chainstate"
+# working_dir = "/dir/to/save/chainstate" # defaults to: /tmp/stacks-node-[0-9]
 rpc_bind = "0.0.0.0:20443"
 p2p_bind = "0.0.0.0:20444"
 bootstrap_node = "02196f005965cebe6ddc3901b7b1cc1aa7a88f305bb8c5893456b8f9a605923893@seed.mainnet.hiro.so:20444,02539449ad94e6e6392d8c1deb2b4e61f80ae2a18964349bc14336d8b903c46a8c@cet.stacksnodes.org:20444,02ececc8ce79b8adf813f13a0255f8ae58d4357309ba0cedd523d9f1a306fcfb79@sgt.stacksnodes.org:20444,0303144ba518fe7a0fb56a8a7d488f950307a4330f146e1e1458fc63fb33defe96@est.stacksnodes.org:20444"
+prometheus_bind = "0.0.0.0:9153" 
 
 [burnchain]
 chain = "bitcoin"
 mode = "mainnet"
-peer_host = "bitcoind.stacks.co"
-username = "blockstack"
-password = "blockstacksystem"
-rpc_port = 8332
```

Comment:
Was this meant to be deleted?

Reply:
it was - along with teh user/pass. 
the reasoning is that it simply isn't needed for running a follower, so i felt it better to remove entirely. 

using the config in this PR still produces a syncing follower:
```
{
  "peer_version": 402653194,
  "pox_consensus": "49e7320b6a861db088f88bcb5ae3dec2da9b9349",
  "burn_block_height": 666075,
  "stable_pox_consensus": "c953d99040d979b0d7848e9a42c15eacb323ce2b",
  "stable_burn_block_height": 666068,
  "server_version": "stacks-node 0.0.1 (:, release build, linux [x86_64])",
  "network_id": 1,
  "parent_network_id": 3652501241,
  "stacks_tip_height": 11,
  "stacks_tip": "f60f0b54a8e07089975f75cd14ef02d8c614d2518c3ed4621bb2b126ca9ea4dc",
  "stacks_tip_consensus_hash": "ecacb0749f8cba8fd5515fd8af8549b4099e352b",
  "genesis_chainstate_hash": "74237aa39aa50a83de11a4f53e9d3bb7d43461d1de9873f402e5453ae60bc59b",
  "unanchored_tip": null,
  "unanchored_seq": null,
  "exit_at_block_height": null,
  "node_public_key": "03e00cff47e2b96f9c635db88188087c53470e7cfee400f23a34bb443aaf3a95ba",
  "node_public_key_hash": "2d366c1238ee2feafa833f795442a38e8752615f",
...
```

Reply:
I'll actually have another commit to push here - give me a few to verify some changes

Reply:
Interesting! So is it only used by miner's to verify transactions it's submitted?

Reply:
afc9c32648e81e3ab3a9073a070aead2ae221d70

Reply:
that's what i found - i accidentally had the wrong RPC port in a follower and it sill synced as expected. 
that mistake led me to look at the other fields in our example configs for possible removal. 
The other interesting thing is that as long as the bitcoin node uses default ports, you don't need to specify those either.

Reply:
i was able to also remove the bootstrap_node setting, for mainnet since there are hardcoded values there. 
for testnet, we do not have bootstrap nodes hardcoded yet (since the network being used is krypton, whereas previously it was xenon which does have hardcoded bootstrap_node). 


i debated on adding a link to the docs for the config settings, ultimately i left it out though. 
i think longterm, we may want to add something either to the binary (`--help`) or some source of truth document that details all the config options.